code/__defines/misc.dm

```diff
@@ -141,14 +141,14 @@
 #define PROG_OFFICE  	"Office Work"
 #define PROG_COMMAND  	"Command"
 #define PROG_SUPPLY  	"Supply and Shuttles"
-#define PROG_ADMIN  	"NTNet Administration"
+#define PROG_ADMIN  	"EXONET Administration"
```

Comment:
I almost feel like the actual name of the network should be a map-level var, with 'Network' used in general. Always bugged me that NTNet and NTOS were hard-baked into things and couldn't be easily removed or changed.

Not really a note for this PR, more one for the future.